cmd/root.go

```diff
@@ -183,3 +186,34 @@ func generateEncodedRandomPassword(s int) (string, error) {
 	pw := base64.URLEncoding.EncodeToString(b)
 	return base64.URLEncoding.EncodeToString([]byte(pw)), nil
 }
+
+func clientForGroupVersionKind(pool dynamic.ClientPool, disco discovery.DiscoveryInterface, gvk schema.GroupVersionKind, namespace string) (*dynamic.ResourceClient, error) {
```

Comment:
seems that you are not using these functions any more, you can delete them

Reply:
No this func is still being used by others

Reply:
I moved it to root.go because they're being used, yep